### PR TITLE
Update Danish translation

### PR DESCRIPTION
**Pull request by scootergrisen**

### CODE REVIEW

File: runtime/lang/menu_da.latin1.vim [hunk 1]
```diff
@@ -1,3 +1,3 @@
 " Menu Translations:	Danish for iso-8859-1 encoding
 
-source <sfile>:p:h/menu_da.utf-8.vim
+source <sfile>:p:h/menu_da.utf-8.vim
```
Comment (k-takata) [inline runtime/lang/menu_da.latin1.vim:3]: Is this change needed?

Comment (scootergrisen) [reply runtime/lang/menu_da.latin1.vim:3]: No i guess not.
The other files does have a newline at the end it seems.

Comment (scootergrisen) [reply runtime/lang/menu_da.latin1.vim:3]: By other files i mean translations.

**Comment by brammool** [comment]
I'll include it, thanks.
